projects/bp-gallery/.eslintrc

```diff
@@ -25,7 +25,7 @@
   "plugins": ["react", "react-hooks", "@typescript-eslint", "prettier", "unused-imports"],
   "rules": {
     "no-alert": "warn",
-    "prettier/prettier": "warn",
+    "prettier/prettier": ["warn", { "endOfLine": "auto" }],
```

Comment:
Because half the project has "wrong" line endings according to prettier and changing them wont actually commit them so I have the same problem the next time I clone the project. I don't know a better way to do this except putting it in my user config instead of the workplace config, but apparently others have the same prettier warnings and just ignore them. Often there are so many prettier warnings that I can't find the actual warnings and errors of the linter

Reply:
this is apparently a windows / linux thing where I'm getting thousands of "remove CR" warnings

Reply:
I have opened  #193 for later investigation